source/js/_module.js

```diff
@@ -0,0 +1,238 @@
+    {"name": "Asus ZenFone 8", "price": "$900.00", "lowerPrice": "1140.00", "photo": "3.jpg"},
+    {"name": "Nokia 5.4", "price": "$240.00", "lowerPrice": "", "photo": "4.jpg"},
+  
+    {"name": "Apple iPhone 12 Pro Max", "price": "$1760.00", "lowerPrice": "", "photo": "iphone1.jpg"},
+    {"name": "Apple iPhone 12", "price": "$990.00", "lowerPrice": "1000.00", "photo": "iphone2.jpg"},
+    {"name": "Apple iPhone SE", "price": "$560.00", "lowerPrice": "", "photo": "iphone3.jpg"},
+    {"name": "Apple iPhone 12 Pro", "price": "$1920.00", "lowerPrice": "", "photo": "iphone4.jpg"},
+    {"name": "Apple MacBook Air 13", "price": "$1500.00", "lowerPrice": "1560.00", "photo": "mac1.jpg"},
+    {"name": "Apple MacBook Pro 13\" A2251 Retina", "price": "$3230.00", "lowerPrice": "", "photo": "mac2.jpg"},
+    {"name": "Apple MacBook Pro 13\" M1", "price": "$1870.00", "lowerPrice": "", "photo": "mac3.jpg"},
+    {"name": "Apple MacBook Pro 16\"", "price": "$4100.00", "lowerPrice": "", "photo": "mac4.jpg"},
+  
+    {"name": "Xiaomi Redmi Note 9", "price": "$1340.00", "lowerPrice": "$1370.00", "photo": "x1.jpg"},
+    {"name": "Xiaomi Redmi Note 10 Pro", "price": "$220.00", "lowerPrice": "230.00", "photo": "x2.jpg"},
+    {"name": "Xiaomi Redmi 9A", "price": "$100.00", "lowerPrice": "", "photo": "x3.jpg"},
+    {"name": "Xiaomi Mi 11 Lite", "price": "$230.00", "lowerPrice": "", "photo": "x4.jpg"},
+    {"name": "OPPO Reno5 Lite", "price": "$210.00", "lowerPrice": "", "photo": "o1.jpg"},
+    {"name": "OPPO A15s", "price": "$120.00", "lowerPrice": "", "photo": "o2.jpg"},
+    {"name": "OPPO A53", "price": "$110.00", "lowerPrice": "", "photo": "o3.jpg"},
+    {"name": "OPPO A52", "price": "$190.00", "lowerPrice": "", "photo": "o4.jpg"},
+  
+    {"name": "Microsoft Surface Laptop 3 - 13.5\"", "price": "$1260.00", "lowerPrice": "", "photo": "note1.jpg"},
+    {"name": "Microsoft Surface Book 3", "price": "$1150.00", "lowerPrice": "", "photo": "note2.jpg"},
+    {"name": "Microsoft Surface Pro 7 Platinum", "price": "$987.00", "lowerPrice": "", "photo": "note3.jpg"},
+    {"name": "Microsoft Surface Laptop GO", "price": "$1320.00", "lowerPrice": "", "photo": "note4.jpg"},
+    {"name": "Xiaomi Mi RedmiBook 14\"", "price": "$1230.00", "lowerPrice": "", "photo": "note5.jpg"},
+    {"name": "Xiaomi Mi Notebook Pro 15.6", "price": "$1110.00", "lowerPrice": "", "photo": "note6.jpg"},
+    {"name": "Xiaomi Mi RedmiBook 16\"", "price": "$1140.00", "lowerPrice": "", "photo": "note7.jpg"},
+    {"name": "Xiaomi Mi RedmiBook 13\"", "price": "$1340.00", "lowerPrice": "", "photo": "note8.jpg"}
+]
+let addIdNumberAndFlag = (arrayData)=>{
+    for (let i = 0; i <arrayData.length; i++){
+        arrayData[i].idIndex = i;
+        arrayData[i].inCart = false;
+    }
+} 
+addIdNumberAndFlag(phonesArr);
+let newPhones = [];
+let renderItems = (arrayPhone) => {
+    
+    template = ['','','','','','']; // ??????? 2 дня потратил 
```

Comment:
dont write comment especially in russian

Reply:
done